### PR TITLE
Update README with correct dependencies.

### PR DESCRIPTION
**Pull request by hameerabbasi**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ provider::
 
     [dependencies]
     ndarray = { version = "0.13.0", features = ["blas"] }
-    blas-src = { version = "0.2.0", default-features = false, features = ["openblas"] }
+    blas-src = { version = "0.4.0", default-features = false, features = ["openblas"] }
```
Comment (LukeMathWalker) [inline README.rst]: We should also bump `openblas-src` to `0.7` :pray:

Comment (hameerabbasi) [reply README.rst]: Done.

**Comment by hameerabbasi** [comment]
Sorry about missing this on my first pass, should probably have done a `grep`.

**Comment by LukeMathWalker** [comment]
No worries, I should have thought about this myself when merging the previous PR :sweat_smile:

**Pull request merged**